framework/src/bcs/VectorPenaltyDirichletBC.C

```diff
@@ -0,0 +1,51 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#include "VectorPenaltyDirichletBC.h"
+#include "Function.h"
+
+registerMooseObject("MooseTestApp", VectorPenaltyDirichletBC);
```

Comment:
The object is still registered to the test app. Change this by:

```suggestion
registerMooseObject("MooseApp", VectorPenaltyDirichletBC);
```

Reply:
I think I fixed this now and amended my commit with a force-push